transforms/ctl/idt/vendorSupplied/blackmagic_design/IDT.BlackmagicDesign.Blackmagic_Film_WideGamut_Gen5_to_ACES.ctl

```diff
@@ -1,5 +1,5 @@
 
-// <ACEStransformID>urn:ampas:aces:transformId:v1.5:IDT.Blackmagic.BMDFilm_WideGamut_Gen5.a1.v1</ACEStransformID>
+// <ACEStransformID>urn:ampas:aces:transformId:v1.5:IDT.BlackmagicDesign.BMDFilm_WideGamut_Gen5.a1.v1</ACEStransformID>
```

Comment:
Should the **ACEStransformID** `name` simply be `Blackmagic Wide Gamut`? It is how I added in https://github.com/colour-science/colour/pull/844 for now.

The paper is a bit inconsistent as I see the following names:

- **Blackmagic Camera Wide Gamut** Primaries and White Point
- **Blackmagic Wide Gamut** (CIE xy chromaticity diagram)
- convert linear **Blackmagic Wide Gamut** RGB data

Reply:
These were the preferred names supplied to me after some back-and-forth with the Blackmagic team.

Reply:
Thanks @scottdyer!